### ISSUE TITLE
Fails to build target 'pdf' in Debian 'unstable'

### ISSUE THREAD

**Issue by xaosfiftytwo** [dpmb/dpmb#2]
me@medion:~/git/dpmb/dpmb$ make
a2x  -f xhtml -L debian-paketmanagement.txt
a2x  -f epub -L debian-paketmanagement.txt
a2x  -f pdf -L debian-paketmanagement.txt
a2x: ERROR: "dblatex" -t pdf -p "/etc/asciidoc/dblatex/asciidoc-dblatex.xsl" -s "/etc/asciidoc/dblatex/asciidoc-dblatex.sty"   "/home/me/git/dpmb/dpmb/debian-paketmanagement.xml" returned non-zero exit status 1
Makefile:14: recipe for target 'pdf' failed
make: **\* [pdf] Error 1

**Comment by hofmannedv** [comment]
Hi xaosfiftytwo,

current release works perfectly, and builds HTML, ePub, and PDF. You may need to install the packages asciidoc, and dblatex to create a pdf file.

HTH,

Frank

**Comment by xtaran** [comment]
Hi xaosfiftytwo,

any chance that you are running Debian 7 Squeeze or Ubuntu 12.04 LTS Precise? Because it's known that dblatex in these releases is too old to work with the book.

**Comment by xaosfiftytwo** [comment]
Hi guys,

As mentioned in the title, I am running Debian unstable. Plus some packages from experimental. Perhaps it is that combination that triggers the error. I will come back to you when I have some more info.
Thanks for the prompt reaction.

**Comment by xtaran** [comment]
> As mentioned in the title, I am running Debian unstable.

Oh, sorry, didn't notice that there's more information in the title. My fault.

More information would be cool. Especially if we have to expect that it will FTBFS in the future.

Then again, I'm most time building the book on Debian Unstable, too, so I should notice rather soon, too.

**Comment by benediktahrens** [comment]
TL;DR: add 'texlive-lang-german' to the install instructions.

# 

I am using Debian 8 (jessie aka stable), where building the pdf target fails as well:

~/git/dpmb/dpmb$ make pdf
a2x  -f pdf -L debian-paketmanagement.txt
a2x: ERROR: "dblatex" -t pdf -p "/etc/asciidoc/dblatex/asciidoc-dblatex.xsl" -s "/etc/asciidoc/dblatex/asciidoc-dblatex.sty"   "/home/ahrens/git/dpmb/dpmb/debian-paketmanagement.xml" returned non-zero exit status 1
Makefile:14: recipe for target 'pdf' failed
make: **\* [pdf] Error 1

Executing the above command yields
$ "dblatex" -t pdf -p "/etc/asciidoc/dblatex/asciidoc-dblatex.xsl" -s "/etc/asciidoc/dblatex/asciidoc-dblatex.sty"   "/home/ahrens/git/dpmb/dpmb/debian-paketmanagement.xml"
Build the book set list...
Build the listings...
XSLT stylesheets DocBook - LaTeX 2e (0.3.5-2)

Don't know what gentext to create for xref to: "informalfigure" (linkend=fig.ara-suche)
Image 'dblatex' not found
Build debian-paketmanagement.pdf
pdflatex failed
/usr/share/texlive/texmf-dist/tex/generic/babel/babel.sty:299: Package babel Error: Unknown option `ngerman'. Either you misspelled it
/usr/share/texlive/texmf-dist/tex/generic/babel/babel.sty:299: leading text: \ProcessOptions*
/usr/share/texlive/texmf-dist/tex/generic/babel/babel.sty:332: Package babel Error: You haven't specified a language option.
/usr/share/texlive/texmf-dist/tex/generic/babel/babel.sty:332: leading text: ...ry to proceed from here, type x to quit.}
debian-paketmanagement.tex:52: Undefined control sequence \shorthandoff.
debian-paketmanagement.tex:52: leading text: \begin{document}
Unexpected error occured
Error: pdflatex compilation failed

As mentioned at the beginning, 

apt install texlive-lang-german

remedied the problem.

EDIT: removed terminal output line of equal signs that was interpreted as markup

**Comment by xtaran** [comment]
Hi Benedikt,

On Sun, Aug 23, 2015 at 09:46:10AM -0700, Benedikt Ahrens wrote:

> TL;DR: add 'texlive-lang-german' to the install instructions.

Thanks. I was quite sure that I tested to build the book in pbuilder
to verify the build-dependencies. Chances are though that I wasn't
able to finish this test due the gcc5 transition which is currently
happening in Sid.

I though remember that I had to add it for building the book on Travis
CI.

Will soon update the installation instructions accordingly.

Thanks!

```
    Kind regards, Axel
```

## 

/~\  Plain Text Ribbon Campaign                   | Axel Beckert
\ /  Say No to HTML in E-Mail and News            | abe@deuxchevaux.org  (Mail)
 X   See http://www.nonhtmlmail.org/campaign.html | abe@noone.org (Mail+Jabber)
/ \  I love long mails: http://email.is-not-s.ms/ | http://abe.noone.org/ (Web)

**Comment by xtaran** [comment]
Closed #2 by merging pull-request #3.

Actually xmlto seems to be needed, too. Not sure if that's installed by default on the Travis CI VMs.

The book now builds fine inside a clean Debian Sid pbuilder chroot.
